### PR TITLE
Consistently xref properties in ArgumentException

### PR DESCRIPTION
**Pull request by ChrisMaddock**
Properties in the remarks section of this API page were inconsistently fenced/xref'd. I assume there's no problem xref'ing everything!

### CODE REVIEW

File: xml/System/ArgumentException.xml [hunk 1]
```diff
@@ -376,8 +376,8 @@
 |Property|Value|  
 |--------------|-----------|  
 |<xref:System.Exception.InnerException%2A>|The inner exception reference.|  
-|`Message`|The error message string.|  
-|`ParamName`|The parameter name string.|  
+|<xref:System.Exception.Message%2A>|The localized error message string.|  
+|<xref:System.Exception.ParamName%2A>|The parameter name string.|
```
Comment (rpetrusha) [inline xml/System/ArgumentException.xml]: This should be \<xref:System.ArgumentException.ParamName%2A>

Comment (ChrisMaddock) [reply xml/System/ArgumentException.xml]: Thanks @rpetrusha - good spot!

Now you've pointed it out - I've noticed the other xref's are inconsistent in the file. Some reference `<xref:System.ArgumentException.Message%2A>`, others `<xref:System.Exception.Message%2A>`. Should they all be referencing ArgumentException, or does it not matter?

**Comment by mairaw** [comment]
CLA seems stuck.

**Comment by ChrisMaddock** [comment]
@rpetrusha Additional changes made - would you be able to take another look?

Thanks for the clarification here. 😄

**Pull request merged**